artifacts/fedora/fedora.go

```diff
@@ -118,6 +129,7 @@ func (f *fedoraGatherer) Gather() ([][]api.Artifact, error) {
 			artifacts = append(artifacts,
 				New(
 					release.Version,
+					release.Arch,
```

Comment:
@0xFelix Do you think it's enough? or we want to have array of artifacts per architecture?

Reply:
I'm asking here because we had an open discussion about it in https://github.com/kubevirt/containerdisks/pull/81#discussion_r1523553452. I left similar question about it.

Reply:
I don't think it is correct right now. It will create two fedora artifacts with the same name and different architectures. We need one artifact which holds both architectures. I was thinking of a Gatherer, which is able to gather multiple architectures per found release/artifact.

See (it will break here) https://github.com/kubevirt/containerdisks/blob/e32b1fda9dcc8697c9425478dfcbf9fb7975dc86/cmd/medius/common/registry.go#L95

Also, you should make `additionalLabels` configurable per architecture, so we could provide different instancetypes/preferences per architecture in the future.

Reply:
@0xFelix Thanks. So we'll have in one entry two Fedora artifacts with different architectures?
```
Entry: [0x1400064a2d0 0x1400064a320]
Artifact: &{39 x86_64 Cloud 0x10763f6e0 map[INSTANCETYPE_KUBEVIRT_IO_DEFAULT_INSTANCETYPE:u1.small INSTANCETYPE_KUBEVIRT_IO_DEFAULT_PREFERENCE:fedora]}
Artifact: &{39 aarch64 Cloud 0x10763f6e0 map[INSTANCETYPE_KUBEVIRT_IO_DEFAULT_INSTANCETYPE:u1.small INSTANCETYPE_KUBEVIRT_IO_DEFAULT_PREFERENCE:fedora]}
```
This way we can push an image index of Fedora 39 with amd64/arm64 image manifests.